rsocket-core/src/main/java/io/rsocket/RSocketFactory.java

```diff
@@ -395,7 +395,6 @@ public ClientRSocketFactory fragment(int mtu) {
 
     /** @deprecated this is deprecated with no replacement. */
     public ClientRSocketFactory errorConsumer(Consumer<Throwable> errorConsumer) {
-      connector.errorConsumer(errorConsumer);
       return this;
     }
```

Comment:
Let's mention in the Javadoc that as of 1.0 this method is a no-op.

Reply:
fixed